src/main/kotlin/org/partiql/scribe/sql/RexConverter.kt

```diff
@@ -79,6 +79,9 @@ public open class RexConverter(
     }
 
     override fun visitRexOpTupleUnion(node: Rex.Op.TupleUnion, ctx: StaticType): Expr {
+        if (node.args.isEmpty()) {
```

Comment:
To address the comment in the other PR https://github.com/partiql/partiql-scribe/pull/49#discussion_r1629710740, the previous logic was moved to `RexConverter` to ensure that `TUPLEUNION` is not passed an empty list of args.

With the `SELECT *` expansion, if there all columns were excluded, there would not be a Scribe error even if the query was invalid. For example if `tbl` had 3 columns `col1`, `col2` and `col3`

The query `SELECT * EXCLUDE t.col1, t.col2, t.col3 FROM tbl AS t` would result in the following query for each dialect that expands `SELECT *`:

```
SELECT FROM tbl AS t
```

the above query is invalid in SQL, so just preemptively giving an error in Scribe.